libc/test/src/unistd/CMakeLists.txt

```diff
@@ -432,7 +432,7 @@ add_libc_unittest(
 
 add_libc_test(
   getopt_test
-  HERMETIC_TEST_ONLY # Uses libc's own stderr
+  UNIT_TEST_ONLY # set_getopt_state only exposed for __internal__ target.
```

Comment:
The comment makes sense in the context of the review but probably not when reading this line alone.
Maybe a comment above `add_libc_test` saying
"getopt_test requires access to set_getopt_state which is only available for __internal_ targets, hence the use of UNIT_TEST_ONLY instead of HERMETIC_TEST_ONLY"

Reply:
Done